framework/src/modules/token/types.ts

```diff
@@ -14,24 +14,25 @@
 
 import { MethodContext, ImmutableMethodContext } from '../../state_machine';
 import { CCMsg } from '../interoperability/types';
+import { JSONObject } from '../../types';
 
 export type TokenID = Buffer;
 
-export interface ModuleConfig {
-	minBalances: {
-		tokenID: string;
-		amount: string;
-	}[];
-	feeTokenID: string;
-	accountInitializationFee: string;
-	escrowInitializationFee: string;
-}
-
 export interface MinBalance {
 	tokenID: Buffer;
 	amount: bigint;
 }
 
+export interface ModuleConfig {
+	minBalances: MinBalance[];
```

Comment:
could you check if minBalances is still used? if not we can just remove

Reply:
I found it only to be used in tests. So I presume it is safe to remove it?

Reply:
These are my favorite kind of changes. I love removing unused code and making the codebase leaner 😀

Reply:
yes safe to remove as long as test pass =)